light/trust_options.go

```diff
@@ -38,16 +36,13 @@ type TrustOptions struct {
 // ValidateBasic performs basic validation.
 func (opts TrustOptions) ValidateBasic() error {
 	if opts.Period <= 0 {
-		return errors.New("negative or zero period")
+		return ErrNegativeOrZeroPeriod
```

Comment:
Same here, for `TestValidateTrustOptions`.

Reply:
Done